src/test/osd/osd-scrub-repair.sh

```diff
@@ -17,6 +17,12 @@
 source $(dirname $0)/../detect-build-env-vars.sh
 source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
 
+if [ `uname` = FreeBSD ]; then
+    use_bluestore=false
```

Comment:
could you add a comment explaining the connection between bluestore and overwrites on erasure-coded pool? and `s/use_bluestore/have_ec_overwrite/`.